rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/40_search.yml

```diff
@@ -1,8 +1,13 @@
 ---
 setup:
+  - requires:
+      cluster_features: "gte_v8.2.0"
+      reason: "_tsid hashing introduced in 8.13"
```

Comment:
Not really matching, but I suppose this is a best effort without digging into git/github history

Reply:
yes, starting this is a rabbit hole and not the goal of this